Dockerfile

```diff
@@ -2,11 +2,13 @@
 FROM python:3.9
 
 # Set working directory.
-
+WORKDIR /fastapi
+COPY ./requirements.txt /fastapi/requirements.txt
 # Expose port 8000 for your FastAPI server.
-
+EXPOSE 8000
 # Install Python requirements using Pip.
-
+RUN pip install -r requirements.txt
 # Copy your codebase into the Docker container.
-
+COPY ./* /fastapi/
 # Run FastAPI server on the port exposed above.
+CMD [ "uvicorn" ,"main:app",  "--reload" ]
```

Comment:
You need to specify the host address here. Please refer to this [StackOverflow answer](https://stackoverflow.com/a/20778887) for the reasoning behind why this is required.

Reply:
Thanks for suggesting the changes. I have made those changes and pushed the change(as a new commit). Now the PR should be reflecting those changes.